### DISCUSSION TITLE
How to disable bell when loosing focus in tmux?

### DISCUSSION THREAD

**Discussion by davidarny** [LazyVim/LazyVim#3563] [Q&A]
Neovim sends a bell for some reason when working with multiple windows on tmux

How can I disable this behavior?

https://github.com/LazyVim/LazyVim/assets/17799810/9b7d26a4-f750-4242-b14c-9103ce152b44

P.S. I tried to disable `belloff`, `t_vb`, `errorbells` and `visualbell` and also disabled all bells in both tmux and zsh, so I think it's really neovim doing some stuff when loosing focus

**Comment by dpetka2001** [comment]
You could set `vim.opt.eventignore = "comma_separated_events"`, but you should check which events are getting executed on `FocusLost` and if you don't mind them if they don't execute. Perhaps there's a better way to do this, but I'm not aware of it.
